docs/conversion/sakai_23_mysql_conversion.sql

```diff
@@ -679,7 +679,7 @@ update rbc_tool_item_rbc_assoc set toolId='sakai.assignment.grades' where toolId
 CREATE TEMPORARY TABLE temp_uuids AS
   SELECT SUBSTRING_INDEX(SUBSTRING_INDEX(RESOURCE_ID, '/', -2), '/', 1) AS extracted_uuid FROM CONTENT_RESOURCE WHERE RESOURCE_ID LIKE '/group/%/site_icon_image.png';
 
-INSERT INTO SAKAI_SITE_PROPERTY (SITE_ID, PROPERTY_NAME, PROPERTY_VALUE) SELECT extracted_uuid, 'custom_image_url', CONCAT('/access/content/group/', extracted_uuid, '/site_icon_image.png') FROM temp_uuids;
+INSERT INTO SAKAI_SITE_PROPERTY (SITE_ID, NAME, VALUE) SELECT extracted_uuid, 'custom_image_url', CONCAT('/access/content/group/', extracted_uuid, '/site_icon_image.png') FROM temp_uuids;
```

Comment:
@ottenhoff could you please review this? It was added by you here -> https://github.com/sakaiproject/sakai-reference/commit/00a4ad62d02d72a79f3a971345f8eecf56ed3720
Thanks!

Reply:
I'm agree with this change. Thanks!